source/entities/EntityBase.h

```diff
@@ -19,6 +19,7 @@
 #define ENTITYBASE_H
 
 #include "entities/GameEntityType.h"
+#include "render/RenderManager.h"
```

Comment:
Adding a dependency to rendermanager here is a real bad idea. That will make a recompilation of the whole project if you change something in the render manager